docs/source/install.rst

```diff
@@ -1,89 +1,101 @@
-Install Guide
-=============
+Installation Guide
+==================
 
-.. _before_install:
+Recommended Environments
+------------------------
 
-Before installing Chainer
--------------------------
+We recommend these Linux distributions.
 
-We recommend these platforms.
-
-* `Ubuntu <http://www.ubuntu.com/>`_ 14.04 LTS 64bit
+* `Ubuntu <http://www.ubuntu.com/>`_ 14.04/16.04 LTS 64bit
 * `CentOS <https://www.centos.org/>`_ 7 64bit
 
-Chainer is supported on Python 2.7.6+, 3.4.3+, 3.5.1+.
-Chainer uses C++ compiler such as g++.
-You need to install it before installing Chainer.
-This is typical installation method for each platform::
+Chainer might also run on Windows and macOS platforms, although we do not guarantee that Chainer run correctly (especially with CUDA support).
 
-  # Ubuntu 14.04
-  $ apt-get install g++
+The following versions of Python can be used: 2.7.6+, 3.4.3+, 3.5.1+, and 3.6.0+.
 
-  # CentOS 7
-  $ yum install gcc-c++
+.. note::
 
-If you use old ``setuptools``, upgrade it::
+   We are testing Chainer automatically with Jenkins, where all the above *recommended* environments are tested.
+   We cannot guarantee that Chainer works on other environments, even if Chainer looks running correctly.
 
-  $ pip install -U setuptools
 
 
-Install Chainer
----------------
+Dependencies
+------------
 
-Chainer depends on these Python packages:
+Before installing Chainer, we recommend to upgrade ``setuptools`` if you are using an old one::
 
-* `NumPy <http://www.numpy.org/>`_ 1.9, 1.10, 1.11
-* `Six <https://pythonhosted.org/six/>`_ 1.9
+  $ pip install -U setuptools
 
-Caffe model support
+The following Python packages are required to install Chainer.
+The latest version of each package will automatically be installed if missing.
 
-* `Protocol Buffers <https://developers.google.com/protocol-buffers/>`_
-* protobuf>=3.0.0 is required for Py3
+* `NumPy <http://www.numpy.org/>`_ 1.9, 1.10, 1.11, 1.12
+* `Six <https://pythonhosted.org/six/>`_ 1.9.0 or higher
```

Comment:
Write "1.9.0+" to align with the notation for other packages (or the other way round).